docs/INSTALL.md

```diff
@@ -42,8 +42,7 @@ cd mmdetection
 d. Install mmdetection (other dependencies will be installed automatically).
 
 ```shell
-pip install mmcv
-python setup.py develop  # or "pip install -v -e ."
+pip install -v -e .
```

Comment:
I test the installation in a new environment and find that `pip install cython` is necessary before installing mmdet, otherwise a `ModuleNotFoundError` error will be raised.

Reply:
I added a step where the user runs `pip install -r requirements/build.txt`. This will ensure that they have all build requirements before installing mmdetection. That should resolve the issue.